questions.md

```diff
@@ -0,0 +1,4 @@
+# Questions regarding pull requests
+
+    * Do we have to always create a pull request on a branch ?
```

Comment:
Yes. And 99% of the time you are working on a branch anyway (master is a branch too!)